README.md

```diff
@@ -1,6 +1,43 @@
 ![GitHub Logo](/logo.png)
 
 # CloudGraphs.jl
-Repo for the CloudGraphs project, ongoing project on http://semisortedblog.wordpress.com, check it out! :) 
+
+[![Build Status][build-img]][build-url]
+[![CloudGraphs][cg-badge-v0.5]][cg-pkg-v0.5]
+[![CloudGraphs][cg-badge-v0.6]][cg-pkg-v0.6]
```

Comment:
The bad printing on the rendered readme should clear once the package shows up in pkg.julialang.org.